docs/howto/manage-web-app-charms/use-web-app-charm.rst

```diff
@@ -432,3 +432,47 @@ the HTTPS URL. You can access the HTTPS URL of your app with a command like:
 To obtain a TLS certificate signed by a trusted certificate authority (CA)
 using the ACME protocol, use the `LEGO <https://charmhub.io/lego>`_
 charm instead.
+
+
+Run a worker or scheduler process in your workloads
+---------------------------------------------------
+
+You can run extra processes in the workload charm that receive the same environment
+variables as the main app. These services must be added to the
+```rockcraft.yaml`` file under the ``services`` section. If the service name
+ends with ``-worker`` it will run in all units of the app,
+while if the name ends with ``-scheduler`` it will run in only one unit.
```

Comment:
```suggestion
You can run extra processes in the workload charm that receive the same environment
variables as the main app. These services must be added to the
project file as entries of the ``services`` key. If the service name
ends with ``-worker`` it will run in all units of the app,
while if the name ends with ``-scheduler`` it will run in only one unit.
```